manifests/osfamily/debian.pp

```diff
@@ -47,7 +43,6 @@
       } else {
```

Comment:
This whole if/else block can be collapsed to the single case. I'd reduce the comment to explain why we don't use the cert/key.